### DISCUSSION TITLE
关于你们提供的软件下载的限制说明

### DISCUSSION THREAD

**Discussion by watchingfun** [natfrp/wiki#348] [想法] [answered]
下载服务限制多线程下载无可厚非，我可以理解。只是希望你们在下载服务哪里加一条说明：“禁止多线程下载工具下载”。我第一次下载时被ban5分钟，后面一段时间下载时又被ban了，主要有时候可能急急急，突然被ban就很😢

**Comment by fakeboboliu** [comment]
您好，

因为存在 DDoS 攻击，我们限制了软件下载 CDN 的请求频率，目前的限制是 5 个请求 / 5 分钟。
我们的软件足够小，这样的限制即使在您使用多线程下载时也应当足够了。

如果您需要一次下载 5 个以上的文件，我们目前没有想到需要这样做的合理场景。
如果您有，请告诉我们，我们将对限制做出优化或此后直接归档提供您需要的文件。

[a comment by an automated account is not shown]
